src/style.css

```diff
@@ -1 +1,51 @@
-/* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: flex-start;
+}
+
+.stars__star {
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. Here you could specify a fallback font-family for `.stars__star` if you have a custom font applied.